zoo-css-challenge/style.css

```diff
@@ -30,7 +34,7 @@ section {
 }
 
 img {
-  border: 5px solid #8075ff;
+  border: none;
```

Comment:
Alternatively, we can delete the `img` rule since `img` elements don't have a border anyway